app/src/main/res/values-hi-rIN/strings.xml

```diff
@@ -28,7 +28,7 @@
     <string name="bloodpressureentry_systolic">सिस्टोलिक</string>
     <string name="bloodpressureentry_diastolic">डायस्टोलिक</string>
     <string name="patientsummary_contentdescription_up_button">वापस जाओ</string>
-    <string name="patientsummary_prescriptions_last_updated">संशोधित हो गया</string>
+    <string name="patientsummary_prescriptions_last_updated">संशोधित हो गया %s</string>
```

Comment:
We should be deleting these strings and adding them as part of the localisation feature instead of modifying them here.

Reply:
Let's keep this for now and delete them when we actually make the change.